meds_interp/meds_logistic_regression.py

```diff
@@ -36,7 +36,13 @@ def score_labels(test_df: pl.DataFrame):
     Returns:
         dictionary of different evaluation metrics.
     """
+    true_labels = test_df["label"]
+    predicted_labels = test_df.get_column("predictions")
```

Comment:
Generate the predictions using the predicted_probs and something like np.argmax():
https://stackoverflow.com/questions/59431676/how-to-get-predicted-class-label-from-predicted-probabilities-in-one-step